README.md

```diff
@@ -1,7 +1,7 @@
 NewspaperWorks
 ===================================================
 Code:
-[![Build Status](https://travis-ci.org/marriott-library/newspaper_works.svg?branch=master)](https://travis-ci.org/marriott-library/newspaper_works) [![Coverage Status](https://coveralls.io/repos/github/marriott-library/newspaper_works/badge.svg)](https://coveralls.io/github/marriott-library/newspaper_works)
+[![Build Status](https://travis-ci.org/samvera-labs/newspaper_works.svg?branch=master)](https://travis-ci.org/samvera-labs/newspaper_works) [![Coverage Status](https://coveralls.io/repos/github/samvera-labs/newspaper_works/badge.svg)](https://coveralls.io/github/samvera-labs/newspaper_works)
```

Comment:
Coveralls link needs setup?

Reply:
I'm hoping this will "just work" once this is merged to master. That's pretty much what happened before.

Seems like the majority of other `samvera-labs` repos that have [coverage reporting](https://coveralls.io/github/samvera-labs) on Coveralls (` valkyrie-sequel`, `dog_biscuits` etc) don't have any explicit `coveralls.yml` file.